docs/Board - ColibriRace.md

```diff
@@ -0,0 +1,126 @@
+# Board - Colibri RACE
+
+The Colibri RACE is a STM32F3 based flight control designed specifically to work with the TBS POWERCUBE multi rotor stack. 
+
+## Hardware Features:
+* STM32F303 based chipset for ultimate performance
+* PPM, SBUS, DSM, DSMX input (5V and 3.3V provided over internal BUS). No inverters or hacks needed.
+* 6 PWM ESC output channels (autoconnect, internal BUS)
+	* RGB LED strip support incl. power management
+	* Extension port for GPS / external compass / pressure sensor
+	* UART port for peripherals (Blackbox, FrSky telemetry etc.)
+	* Choose between plug & play sockets or solder pads for R/C and buzzer
+	* 5V buzzer output
+	* MPU6500 new generation accelerometer/gyro
+	* 3x status LED (DCDC pwr/ 3.3V pwr/ status)
+	* Battery monitoring for 12V, 5V and VBat supply
+	* Size: 36mmx36mm (30.5mm standard raster)
+	* Weight: 4.4g
+
+	For more details please visit:
+	http://www.team-blacksheep.com/powercube
+
+## Serial Ports
+
+	| Value | Identifier   | Board Markings | Notes                                     |
+	| ----- | ------------ | -------------- | ------------------------------------------|
+	| 1     | VCP          | USB PORT       | Main Port For MSP                         |
+	| 2     | USART1       | FREE PORT      | PC4 and PC5(Tx and Rx respectively)       |
+	| 3     | USART2       | PPM Serial     | PA15                                      |
+	| 4     | USART3       | GPS PORT       | PB10 and PB11(Tx and Rx respectively)     |
+
+## Pinouts
+
+	Full pinout details are available in the manual, here:
+
+	http://www.team-blacksheep.com/colibri_race
```

Comment:
link will work once the product is live. it's not yet available to end-customers. we are still waiting for it get ready for shipping :)